.github/workflows/job-postcommit-java-validatesrunner-flink.yml

```diff
@@ -0,0 +1,63 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+name: Java VR Flink PostCommit
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+permissions: read-all
+
+jobs:
+  set-properties:
+    runs-on: [self-hosted, ubuntu-20.04]
+    outputs:
+      properties: ${{ steps.test-properties.outputs.properties }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - id: test-properties
+        uses: ./.github/actions/setup-default-test-properties
+  java_vr_flink:
+    needs: set-properties
+    name: Run Flink Java ValidatesRunner
+    runs-on: [self-hosted, ubuntu-20.04]
+    strategy:
+      fail-fast: false
+      matrix:
+        flink-versions: ${{fromJson(needs.set-properties.outputs.properties).JavaTestProperties.FLINK_VERSIONS}}
+        exclude: # exclude other versions, we only need 1.13 for this test
```

Comment:
We should do the same thing as in the other test if we only need 1.13 (use that as the version instead of an exclude list)

Reply:
I recently bumped DataProc to 2.1 which runs flink 1.15. DataProc never supports flink 1.13 (Dataproc 2.0 runs flink 1.12)

Reply:
@Abacn should we just be pinning to 1.15 across both suites? Probably?

Reply:
yeah I would suggest. For Jenkins it is set here: https://github.com/apache/beam/blob/e1cd5c4faf8d8c96491a144a0c686ca083d48d93/.test-infra/jenkins/CommonTestProperties.groovy#L28 we could also define a single test version for Github Action.

Reply:
Hi @damccorm 
So for this workflow, should I use the `1.15 flink version` instead of `1.13`?

Reply:
Yeah - could you actually set a `FLINK_LATEST` property to 1.15 here - https://github.com/apache/beam/blob/9b3b403ae9c071dcef0e801c892eab2203db88df/.github/actions/setup-default-test-properties/test-properties.json#LL17C48-L17C48 - and use that in both workflows?

Reply:
Yeah I can apply the change, just I tested using 1.15 but the test failed, I changed back to 1.13 and the test passed. 🤔

Reply:
@Abacn any ideas? https://github.com/MarcoRob/beam/actions/runs/3622938989/jobs/6108239170

> I recently bumped DataProc to 2.1 which runs flink 1.15. DataProc never supports flink 1.13 (Dataproc 2.0 runs flink 1.12)

Does this actually affect this set of tests? Or are these just running on a local cluster?

Reply:
No, that change does not affect local clusters. Running flink 1.13 on local cluster does not involve DataProc and does not have problem. For test failure not sure the cause, but we can defer the version aligning as future task if bumping version breaks test.

Reply:
Change applied using `Flink Version 1.13`, I applied the following [property](https://github.com/apache/beam/blob/52e377feee3fe3b27ff6aa780ed3901e382006a6/.github/actions/setup-default-test-properties/test-properties.json#L18) as `FLINK_DEFAULT_VERSION`